vms/avm/service_test.go

```diff
@@ -830,7 +939,7 @@ func TestServiceGetTxJSON_OperationTxWithNftxMintOp(t *testing.T) {
 	require := require.New(t)
 
 	env := setup(t, &envConfig{
-		vmStaticConfig: noFeesTestConfig,
```

Comment:
we are not going to upgrade the tx building APIs but should get rid of the `zeroFeeConfig` from unit tests, since with E upgrade activation there won't be zero fees anymore.
This means modifying service UTs to account for non-zero fees (adding inputs and outputs where needed)